resources/frameworks.yaml

```diff
@@ -42,14 +51,15 @@ autosklearn:
     It leverages recent advantages in Bayesian optimization, meta-learning and ensemble construction.
   project: https://automl.github.io/auto-sklearn/
   refs: [http://papers.nips.cc/paper/5872-efficient-and-robust-automated-machine-learning.pdf]
-#  params:
-#    _save_artifacts: ['models', 'debug_as_files']
-#    _n_jobs: 1
+# params:
+#   _save_artifacts: ['models', 'debug_as_files']
+#   _n_jobs: 1
```

Comment:
this and all changes below are irrelevant for this PR.
Please only modify what is related for your framework.